src/m365/teams/commands/channel/channel-set.ts

```diff
@@ -123,6 +142,44 @@ class TeamsChannelSetCommand extends GraphCommand {
 
     return requestBody;
   }
+
+  private async getTeamId(args: CommandArgs): Promise<string> {
+    if (args.options.teamId) {
+      return args.options.teamId;
+    }
+
+    const group: Group = await aadGroup.getGroupByDisplayName(args.options.teamName!);
+
+    if ((group as ExtendedGroup).resourceProvisioningOptions.indexOf('Team') === -1) {
+      throw 'The specified team does not exist in the Microsoft Teams';
+    }
+    else {
+      return group.id!;
+    }
+  }
+
+  private async getChannelId(args: CommandArgs): Promise<string> {
+    if (args.options.id) {
+      return args.options.id;
+    }
+
+    const channelRequestOptions: any = {
+      url: `${this.resource}/v1.0/teams/${encodeURIComponent(this.teamId)}/channels?$filter=displayName eq '${encodeURIComponent(args.options.name as string)}'`,
+      headers: {
+        accept: 'application/json;odata.metadata=none'
+      },
+      responseType: 'json'
+    };
+
+    const res: { value: Channel[] } = await request.get<{ value: Channel[] }>(channelRequestOptions);
+    const channelItem: Channel | undefined = res.value[0];
+
+    if (!channelItem) {
+      throw `The specified channel does not exist in the Microsoft Teams team`;
```

Comment:
Let's change this to `The specified channel does not exist in this Microsoft Teams team`